### ISSUE TITLE
anyone have Pushpa 2 movie hindi + telegu please give me link 

### ISSUE THREAD

**Issue by Mehra21** [pushpa-2-moov-ab/.github#1]
hey, guys i really want to watch new pushpa movie please someone atleast give me the movie 3rd class quality or 360p please someone giveme the movie link i don't have the money to buy ticket (crying emoji 5*)

**Comment by Mehra21** [comment]
watch and download: [https://freepdfspot.com/puspa-2-full-movie-hindi/](https://google.drive.com/t2gdnfs.........)
watch and download: [https://freepdfspot.com/puspa-2-full-movie-hindi/](https://google.drive.com/t2gdnfs.........)

**Comment by Mehra21** [comment]
watch and download: [https://freepdfspot.com/puspa-2-full-movie-hindi/](https://google.drive.com/t2gdnfs.........)
watch and download: [https://freepdfspot.com/puspa-2-full-movie-hindi/](https://google.drive.com/t2gdnfs.........)
watch and download: [https://freepdfspot.com/puspa-2-full-movie-hindi/](https://google.drive.com/t2gdnfs.........)
watch and download: [https://freepdfspot.com/puspa-2-full-movie-hindi/](https://google.drive.com/t2gdnfs.........)
watch and download: [https://freepdfspot.com/puspa-2-full-movie-hindi/](https://google.drive.com/t2gdnfs.........)
watch and download: [https://freepdfspot.com/puspa-2-full-movie-hindi/](https://google.drive.com/t2gdnfs.........)

**Comment by capcut-pro-apk-download** [comment]
Yup. I can provide you with the right thing. Here you go [Download Puspa 2](tinyurl.com/radibankhatey)
